src/base/digest32.h

```diff
@@ -61,54 +59,82 @@ class Digest32
 
     bool isValid() const
     {
-        return m_dataPtr->valid;
+        return m_dataPtr->isValid();
     }
 
     Digest32 &operator=(const Digest32 &other) = default;
     Digest32 &operator=(Digest32 &&other) = default;
 
     operator UnderlyingType() const
     {
-        return m_dataPtr->nativeDigest;
+        return m_dataPtr->nativeDigest();
     }
 
     static Digest32 fromString(const QString &digestString)
```

Comment:
maybe move static function to the bottom of `public:` section.

Reply:
> maybe move static function to the bottom of `public:` section.

Ok. Done.